src/main/java/org/diego/tutorial/car/timers/CarsTimer.java

```diff
@@ -0,0 +1,40 @@
+package org.diego.tutorial.car.timers;
+
+import java.util.List;
+
+import javax.ejb.EJB;
+import javax.ejb.Schedule;
+import javax.ejb.Singleton;
+
+import org.apache.log4j.Logger;
+import org.diego.tutorial.car.model.Car;
+import org.diego.tutorial.car.model.service.CarService;
+
+/**
+ * Class that implements a timer for the operations related to the cars
+ * */
+@Singleton
+public class CarsTimer {
```

Comment:
unit test?

Reply:
would you need @Startup to initialize the singleton?

Reply:
I did some tests on my own, and the timer was running without the need of startup annotation. Should I add it anyway?

Reply:
only if you need an eager initialization (i.r. some data initialization your API need to work properly). Not needed here apparently.

Reply:
Still unit test needed

Reply:
Unit test added